torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -8471,13 +8471,14 @@ def test_ddp_model_diff_num_params_across_ranks(self):
                 group_gloo,
             )
 
-        def _test_output_unused_in_loss(self, module_cls, gradient_as_bucket_view):
+        def _test_output_unused_in_loss(self, module_cls, static_graph):
```

Comment:
oops, look like this test never tested grad_as_bucket_view as we should be. Could we add grad_as_bucket view to DDP ctor below?